frontend/src/App.js

```diff
@@ -79,15 +82,20 @@ function App() {
   const [newslist, setNewslist] = useState([]);
   const [primaryTicker, setPrimaryTicker] = useState("AAPL");
   const [watchlist, setWatchlist] = useState([]);
-  const [bottomState, setBottomState] = useState(true);
+  const [, setBottomState] = useState(true);
   const [showSignOut, setShowSignOut] = useState(false);
+  const [showForgotPassword, setShowForgotPassword] = useState(false);
+  const [showReset, setShowReset] = useState(false);
   const [showAuth, setShowAuth] = useState(false);
+  const [queryString] = useState(window.location.search);
+
+  const user = Cookies.get("user");
 
   useEffect(() => {
     const HTTP_OK = 200;
     const fetchData = async () => {
       let response = await fetch(
-        `http://localhost:6969/quote/${primaryTicker}`
+        `http://localhost:6969/api/markets/quote/${primaryTicker}`
```

Comment:
What do you think about setting `https://localhost:6969/api/` as a `.env` variable called `API_HOST`? That way when this eventually gets deployed we only have to change the value in one spot.

Reply:
Good idea, added in: https://github.com/ngambino0192/stocks/pull/11/commits/70cfe25b69530022d5e38f2c59f1ce4e73dee36f